### PR TITLE
Update the trigger_upload.py script to add push_notifications

### PR DESCRIPTION
**Pull request by sayanchowdhury**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,17 @@ a job manually. If Fedimg is installed properly, the `bin/trigger_upload.py`
 script can be used for this purpose:
 
 ```
-./bin/trigger_upload.py SOME_RAWXZ_URL COMPOSE_ID
+usage: trigger_upload.py [-h] -u URL -c COMPOSE_ID [-p PUSH_NOTIFICATIONS]
```
Comment (dustymabe) [inline README.md]: this should be `[-p]` since there is no arg data that is provided

File: README.md [hunk 2]
```diff
@@ -21,7 +21,17 @@ a job manually. If Fedimg is installed properly, the `bin/trigger_upload.py`
...
+
+Trigger a manual upload process with the specified raw.xz URL
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -u URL, --url URL     .raw.xz URL
+  -c COMPOSE_ID, --compose-id COMPOSE_ID
+                        compose id of the .raw.xz file
+  -p PUSH_NOTIFICATIONS, --push-notifications PUSH_NOTIFICATIONS
```
Comment (dustymabe) [inline README.md]: should be just `[-p]`

File: bin/trigger_upload.py [hunk 3]
```diff
@@ -2,28 +2,40 @@
...
+        required=False, default=False)
+    args = parser.parse_args()
 
-compose_meta = {
-    'compose_id': compose_id
-}
+    return args.url, args.compose_id, args.push_notifications
 
-fedimg.uploader.upload(upload_pool, [url], compose_meta=compose_meta)
+url, compose_id, push_notifications = get_args()
```
Comment (dustymabe) [inline bin/trigger_upload.py]: we might as well make a `def main()` here :)

File: README.md (hunk 1 above)
Comment (sayanchowdhury) [reply README.md]: Actually right now it expects arg data. I would update the script so that it expects no arg data

**Pull request merged**